planner/core/logical_plan_test.go

```diff
@@ -1838,13 +1843,14 @@ func (s *testPlanSuite) TestUnion(c *C) {
 		plan, err := builder.Build(stmt)
 		if tt.err {
```

Comment:
This part can be changed directly to `c.Assert(tt.err, NotNil)`

Reply:
When `tt.err` is false, we should ignore below code. Is it can use `c.Assert` directly?